test/integration/aab_offline_test.go

```diff
@@ -49,6 +49,40 @@ func TestOffline(t *testing.T) {
 				// RFC1918 address that unlikely to host working a proxy server
 				env = append(env, "HTTP_PROXY=172.16.1.1:1")
 				env = append(env, "HTTP_PROXYS=172.16.1.1:1")
+
```

Comment:
the goal of this test should be, we simulate an enviroment that minikube is disconnected from internet,
so `minikube ssh docker pull a_new_image` should not work but everything else should work.

could u please verify, that setting these env variables, will Prevent minikube from downloading anything new ?
```
HTTP_PROXY=172.16.1.1:1"
HTTP_PROXYS=172.16.1.1:1
```
if we set those envs and do 
"minikube ssh"
and then try to "docker pull a_new_image"
that should fail, (to ensure there is no internet)
but kubernetes itself should not fail, since minikube should load the kubernets images from cache

if we add that verification to this test we can be esnure we dont break ppl who use minikube on an aireplane.